### ISSUE TITLE
Make into a PWA

### ISSUE THREAD

**Issue by jgw96** [joshuamorony/permatimer#1]
If there is any interest in making a PWA version of this app id love to help!

**Comment by joshuamorony** [comment]
Yeah for sure, should make an interesting follow-up blog post as well (I'll be publishing a post on this app this week).

Feel free to make any changes you please!

**Comment by jgw96** [comment]
Awesome! Opened a PR [here](https://github.com/joshuamorony/permatimer/pull/2) to make this a PWA.
